QualityControl/lib/services/ccdb/CcdbService.js

```diff
@@ -185,18 +185,32 @@ export class CcdbService {
     };
     const url = `/latest${this._buildCcdbUrlPath(partialIdentification)}`;
 
-    const result = await httpGetJson(this._hostname, this._port, url, { headers });
-    if (result?.objects?.length > 0) {
-      const [qcObject] = result.objects;
-      return {
-        [CCDB_RESPONSE_BODY_KEYS.PATH]: qcObject[CCDB_RESPONSE_BODY_KEYS.PATH],
-        [CCDB_RESPONSE_BODY_KEYS.VALID_FROM]: qcObject[CCDB_RESPONSE_BODY_KEYS.VALID_FROM],
-        [CCDB_RESPONSE_BODY_KEYS.VALID_UNTIL]: qcObject[CCDB_RESPONSE_BODY_KEYS.VALID_UNTIL],
-        [CCDB_RESPONSE_BODY_KEYS.ID]: qcObject[CCDB_RESPONSE_BODY_KEYS.ID],
-      };
-    } else {
-      throw new Error(`Object: ${url} could not be found`);
+    let result = null;
+    try {
+      result = await httpGetJson(this._hostname, this._port, url, { headers });
+    } catch {
+      const errorMessage = this._buildFilterErrorMessage(
+        `Object at url '${url}' and path '${partialIdentification.path}' could not be found.`,
```

Comment:
At this point we do not know that the actual error is that object could not be found. It can be a timeout, a bad service, etc.